pkg/frontend/openshiftcluster_preflightvalidation.go

```diff
@@ -55,6 +55,7 @@ func (f *frontend) preflightValidation(w http.ResponseWriter, r *http.Request) {
 			res := f._preflightValidation(ctx, log, raw, typeMeta.APIVersion, typeMeta.Id)
 			if res.Status == api.ValidationStatusFailed {
 				log.Warningf("preflight validation failed")
+				log.Warningf(to.String(res.Error.Message))
```

Comment:
nit: 

don't need `Warningf` just `Warning` will be fine since we're not using format strings.